README.md

```diff
@@ -105,22 +109,26 @@ Download/Clone the Git repository.
 
 
 ## Setup Interfaces
-Install dependencies for the interfaces, from the root of this repository, run:
+Install dependencies for the interfaces, from the individual interface directories (`interfaces/*`) run:
 
-    npm run install:interfaces
+    npm install
 
 Or to install 1 specific interface's dependencies, run: (where `<interface-name>` is one of `pa`, `aw`, `ho`, `referral`)
 
     npm run install:<interface-name>
 
 ## Setup Services
+Install dependencies for the interfaces, from the individual interface directories (`services/*`) run:
+
+    npm install
+
```

Comment:
This is only required if you want to work on a service in a 'standalone', local way.
It is not the most convenient way and needs some manual adjustments.

Services can/should run within the Docker environment by default. Any other way should only be done "when you know what you're doing".

```suggestion
```